packages/moti/src/core/types.ts

```diff
@@ -205,7 +180,7 @@ export type MotiAnimationProp<Animate> = MotiProps<Animate>['animate']
 export type MotiFromProp<Animate> = MotiProps<Animate>['from']
 export type MotiExitProp<Animate> = MotiProps<Animate>['exit']
 
-type OrDerivedValue<T> = T | DerivedValue<T>
+type OrDerivedValue<T> = T | Omit<DerivedValue<T>, 'set'>
```

Comment:
I was getting compiler errors for certain assignments (e.g. in `Moti.Loop.tsx`). Omitting the `set` method solved it.